src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.html

```diff
@@ -414,6 +414,10 @@
                 <div class="col-12">
                   <button type="button"
                           class="btn btn-light float-right tc_addCapButton"
+                          [disabled]="hasAllCapabilities()"
+                          i18n-tooltip
+                          tooltip="All capabilities are already added."
```

Comment:
`tooltip` seems to not work in a disabled button, in Chrome.
https://getbootstrap.com/docs/4.4/components/tooltips/#disabled-elements

Reply:
@tspmelo The issue here explained:
https://jakearchibald.com/2017/events-and-disabled-form-fields/
```mouse events don't fire when the pointer is over disabled form elements, except in Firefox```

As we are using ngx-bootstrap (not plain bootstrap), it seems appropriate to me to keep tooltip in button and trigger the required events (tested and it works):
```
triggers="pointerenter pointerleave"
```